DailyCodingProblem/DailyCodingProblem/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DailyCodingProblem
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Problem1();
+        }
+
+        static void Problem1()
+        {
+            // Given a list of numbers and a number k, return whether any two numbers from the list add up to k.           
+
+            List<int> listOfNumbers = new List<int>() { 8, 1, 9, 3, 7, 6, 12, 36, 78, 25, 5 };
+
+            Console.Write($"Problem 1:\nPlease enter a number and I'll tell you if any two numbers from the list add up to it: ");
```

Comment:
I think it would be nice to print out the list, the user might want to input different things depending on what is in the list